app/web/components/Alert.tsx

```diff
@@ -20,20 +21,18 @@ export default function Alert({
   children,
   ...otherProps
 }: AlertProps) {
-  const classes = useStyles();
-
   const oldErrorKey = Object.keys(grpcErrorStrings).find(
     (oldError): oldError is ObscureGrpcErrorMessages =>
       children.includes(oldError)
   );
 
   return (
-    <MuiAlert {...otherProps} className={classNames(classes.root, className)}>
+    <StyledAlert {...otherProps} className={classNames(className)}>
```

Comment:
I think you can get rid of the `classNames` function and import and just put the `className` prop here now.

Reply:
done :)